docs/user/interactive_customizing.rst

```diff
@@ -164,6 +174,58 @@ these base ``Dockerfile`` s and add the ``renku``, ``git``, and ``jupyter``
 parts to another base image that you might have.
 
 
+Renku project configurations
```

Comment:
Maybe we can move this section before the Docker section? I imagine people might want to change this more often than they would want to dig into the Docker build itself.